jina/hubble/hubio.py

```diff
@@ -613,19 +613,26 @@ def deploy_public_sandbox(uses: str):
 
         host = None
         try:
-            res = requests.get(
+            json = requests.get(
```

Comment:
can we have a different name, it can be confused with the package

Reply:
maybe json_response?

Reply:
addressed